### DISCUSSION TITLE
Why are the text labels of the edit extension missing?

### DISCUSSION THREAD

**Discussion by kimku99** [datenstrom/community#582] [Ask a question] [answered]
It seems that all the text labels of the Edit extension are missing, only the input box appears.

Or only I have this problem.
If only I have this problem, it should be my problem during the update

**Comment by markseuffert** [comment]
Do you have more information? Have a screenshot? Does this also happen in a fresh installation?

**Comment by kimku99** [comment]
Here is the screenshot. I haven't tested the fresh installation yet, do you have the same situation?

![yellow-edit-welcome screenshot](https://user-images.githubusercontent.com/59055424/102972902-c7f1df80-4536-11eb-9143-82af054e3042.png)

**Comment by kimku99** [comment, marked as answer]
Sorry! I checked my template. This is my own problem.
